### DISCUSSION TITLE
Deprecation directive appears to render incorrectly

### DISCUSSION THREAD

**Discussion by pushfoo** [pradyunsg/furo#596] [Potential Issue] [answered]
```
python: Python 3.9.2
furo==2022.6.4.1
Sphinx==5.1.1
```
I attempted to use [sphinx's `.. deprecated::` directive](https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-deprecated). However, it appears to render as italic text in front of normal paragraph text instead of an admonition box:
![image](https://user-images.githubusercontent.com/36696816/206853282-2d7dd5d7-6eb3-4df6-b289-8bc20d2a4466.png)

[The `.. warning::` directive](https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-warning) appears to work as expected:
![image](https://user-images.githubusercontent.com/36696816/206853406-1e02a638-3a67-480f-aef4-b9ff841e3cc2.png)

My docstring source:
```py
    """
    Get an image from the screen, using the top left as the origin.

    .. deprecated:: 2.7

                           Use :py:func:`arcade.get_screenshot` or
                           :py:meth:`~Window.get_screenshot` instead to ensure your
                           project continues working with future arcade versions!

    :param int x: Left edge of capture area
    :param int y: Top edge of capture area
    :param int width: Width of image. Leave blank to grab everything right of x
    :param int height: Height of image. Leave blank to grab everything below y
    :returns: A Pillow Image
    :rtype: PIL.Image.Image
    """
```
It looks like the `deprecated` directive is not being classified as an admonition when it should be:
![image](https://user-images.githubusercontent.com/36696816/206853548-c07ded4c-cbb1-46c3-a607-31281784d9c5.png)
![image](https://user-images.githubusercontent.com/36696816/206853596-8d0d3864-ab2b-48e5-857b-a162c947bb21.png)

**Comment by dgw** [comment, marked as answer]
I don't believe it's supposed to be classed as an admonition. The Sphinx docs you linked say it's "similar to versionchanged", which is not an admonition either. Furo renders `.. deprecated::` almost identically to the other themes I've used or seen, including the main Python documentation ([example](https://docs.python.org/3.10/library/abc.html#abc.abstractclassmethod)).

Note that Sphinx themes generally do not modify the HTML output of the documentation itself. When Sphinx loads a theme, it fills placeholders in the layout with the generated HTML, and the layout includes the theme's styles/scripts to define its appearance/behavior.

**Reply by pushfoo** [reply]
Thank you for your prompt response and for the example from Python's doc. I think I misremembered a custom `:deprecated:` directive in a project I contribute to as that of default sphinx.
